recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -3,46 +3,40 @@
 {% set sha2563k = "b4027e018188a1ea29ab3fb5374087bac569c290335a1d5cde8356b446069435" %}
 
 package:
-  name: pybufr-ecmwf  # [py2k]
-  name: pybufr-ecmwf3  # [py3k]
+  name: pybufr-ecmwf
   version: {{ version }}
 
 source:
-  fn: pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
-  fn: pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
+  fn: pybufr-ecmwf-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
   url: https://pypi.io/packages/source/p/pybufr-ecmwf3/pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
   sha256: {{ sha256 }}  # [py2k]
   sha256: {{ sha2563k }}  # [py3k]
 
 build:
-  skip: true  # [win]
+  skip: True  # [win]
   number: 0
-  script:
-    - python clean.py
```

Comment:
I am unclear on the use of `clean.py` but I guess it is to clean from a previous build. If so there is no need for it here.